package.json

```diff
@@ -126,8 +126,8 @@
     "@storybook/manager-webpack5": "6.5.15",
     "@storybook/react": "6.5.15",
     "@storybook/theming": "6.5.15",
-    "@swc-node/register": "1.6.1",
-    "@swc/core": "1.3.30",
+    "@swc/cli": "0.1.62",
+    "@swc/core": "1.3.44",
```

Comment:
last swc/core version that works with swc/helpers v0.4.x, beyond this we need to migrate all v9 packages dep to helpers 0.5

Reply:
from API and semver, swc should have bumped to 2.0 as switching to helper 0.5 its a breaking change. welp

Reply:
we should trigger an issue for this in swc

Reply:
that's definitely the right approach but it looks like that train has already left the station. ATM core latest version is `1.3.71`, based on their approach I don't think they would be willing to do anything about it.